### ISSUE TITLE
Can't stop remote:true forms from sending request

### ISSUE THREAD

**Issue by brandoncordell** [rails/rails#31454]
### Steps to reproduce

Create a simple with `form_with` (defaulting to `remote: true`).

In a javascript file attempt to stop the xhr request from sending.

```
$(document).on('ajax:beforeSend', '#newEcardProgram', function (event) {
  // none of these work
  event.stopPropagation();
  event.preventDefault();
  return false;
});
```

### Expected behavior
Returning false from `ajax:beforeSend` and `ajax:send` should stop the event (and the form) from completing it's request.

The callbacks are running as I'm able to see `console.log` output from each of them.

### Actual behavior
The form submits no matter how I try to stop it

### System configuration
**Rails version**: 5.1.4

**Ruby version**: 2.4.2p198

**Comment by brandoncordell** [comment]
Returning false from `ajax:beforeSend` is stopping Rails from including the CSRF Token on the request though. So while it sends the request it fails everytime.

**Comment by y-yagi** [comment]
Thanks for your create issue.  This already fixed by #29127. 
Please wait for release next version. Thanks!
